src/psij/__init__.py

```diff
@@ -26,8 +26,12 @@
     'InvalidJobException', 'UnreachableStateException', 'Export', 'Import'
 ]
 
+if os.environ['PSIJ_LOG_LEVEL']:
```

Comment:
If PSIJ_LOG_LEVEL is not defined, that line will throw an exception and prevent any code using psij from working.

I think what you want to check is `if 'PSIJ_LOG_LEVEL' in os.environ`:

Reply:
Changed, thanks for the suggestion